### PR TITLE
966 Transform URL from other galleries into kodadot

### LINKED ISSUE #966: Form to convert Singular url to KodaDot
User will input singular url and will get KodaDot url
- profile
- collection
- nft detail

### PR DESCRIPTION
**Pull request by vikiival**
- :top: updated can-i-use
- :tada: url transformer is working correclty
- :white_check_mark: updated tests for url transformer
- :lipstick: Disabled input can accept readonly prop
- :rocket: added Tranform component
- :earth_americas: added translation for the transform
- :zap: added translations, added transform to navbar, added ?url param

Thank you for your contribution to the KodaDot NFT gallery, 
we really appreciate your contribution!

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### Before submitting this PR, please make sure:
- [x] Your code builds clean without any erros or warnigns
- [x] You've posted screenshot of demonstrated change in this PR
- [x] Merged recent default branch, **main** and you have no conflicts
- [x] Didn't break any original functionality 

### Optional
- [ ] You've tested it on mobile
- [ ] Are there any edge cases? Name if any 

### What's new? (may be part of changelog)
- closes #966

Screenshot 2021-11-29 at 17 08 29

### CODE REVIEW

File: src/views/Transform.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="columns mb-6">
+    <div class="column is-6 is-offset-3">
```
Comment (roiLeo) [inline src/views/Transform.vue:3]: I would change it by 
```html
<div class="columns is-centered">
  <div class="column is-half">
```

File: src/views/Transform.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      <section>
```
Comment (roiLeo) [inline src/views/Transform.vue:4]: You could try it with:
```html
<section class="section">
```
to add a gutter

**Review by roiLeo — CHANGES_REQUESTED** [review]

File: src/views/Transform.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import { exist } from '@/components/rmrk/Gallery/Search/exist'
+
+@Component({
+  components: {
+    BasicInput: () => import('@/components/shared/form/BasicInput.vue'),
+    DisabledInput: () => import('@/components/shared/DisabledInput.vue'),
+  },
+})
+export default class Transform extends Vue {
+  protected url = ''
```
[the remark this answers is not in the document]
Comment (vikiival) [reply src/views/Transform.vue:42]: It's automatically given from the default value. If I add `string` eslint will shout

File: src/views/Transform.vue [hunk 4]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="columns mb-6">
+    <div class="column is-6 is-offset-3">
+      <section>
+        <p class="title is-size-3">
+            $t('Transform')
```
Comment (roiLeo) [inline src/views/Transform.vue:6]:
```suggestion
          {{ $t('Transform') }}
```
oups! my bad.

**Review by roiLeo — CHANGES_REQUESTED** [review]

**Pull request merged**